tests/test_integration_employee.py

```diff
@@ -110,20 +111,14 @@ def test_create_employee(self):
             },
         )
 
-    @patch('mora.service.cpr.get_citizen')
-    def test_cpr_lookup(self, mock_get_citizen):
+    def test_cpr_lookup_prod_mode_false(self):
         # Arrange
-
-        mock_get_citizen.return_value = {
-            'fornavn': 'First',
-            'mellemnavn': 'Middle',
-            'efternavn': 'Last'
-        }
+        settings.PROD_MODE = False
```

Comment:
You should either mock this or use `util.override_settings()` — sooner or later, you'll break some other test.